src/main/java/com/ipiecoles/java/java350/model/Employe.java

```diff
@@ -113,7 +148,18 @@ else if (this.performance == null || Entreprise.PERFORMANCE_BASE.equals(this.per
     }
 
     //Augmenter salaire
-    //public void augmenterSalaire(double pourcentage){}
+    public void augmenterSalaire(double pourcentage) throws IllegalArgumentException{
+        // Si pourcentage négatif,
+        // Alors ne pas toucher au salaire de base et renvoyer une exception
+        if (pourcentage >= 0) {
+            this.salaire += pourcentage/100 * this.salaire;
```

Comment:
Attention à la `NullPointerException` si `this.salaire` est `null`. Test à ajouter absolument